test/testdrive/mzcompose.py

```diff
@@ -180,7 +180,7 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
             # without mzcompose
             for file in args.files:
                 c.run_testdrive_files(
-                    f"--junit-report={junit_report}",
+                    # f"--junit-report={junit_report}",
```

Comment:
Intentional?

Reply:
Nope! Was trying to run testdrive with `--rewrite-results` and it would complain about being incompatible with `--junit-report`. Thanks for catching!